api/cookie_api_handler.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import "net/http"
+
+const JWTAuthorizationHeaderName = "X-JWT-Authorization"
+
+// CookieAPIHandler set JWT header with JWT cookie value
+func CookieAPIHandler(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// read cookie (no need to validate, this will be done in the API
+		cookie, err := r.Cookie(JWTCookieName)
+		if err == nil {
+			r.Header.Set(JWTAuthorizationHeaderName, cookie.Value)
```

Comment:
I realize this is existing behaviour.  But per https://jwt.io/introduction this is the wrong header to use:
> Whenever the user wants to access a protected route or resource, the user agent should send the JWT, typically in the Authorization header using the Bearer schema. The content of the header should look like the following:
>
> ```
> Authorization: Bearer <token>
> ```

We can (and I think should) change it here, and eventually on the client side too.  This should be in a separate PR, so I opened #1362.

Reply:
Agree, I just wrote a test today for the JWT auth, then I realized I need to set a non-standard header.